samples/api/dynamic_uniform_buffers/dynamic_uniform_buffers.cpp

```diff
@@ -340,7 +340,6 @@ void DynamicUniformBuffers::prepare_pipelines()
 
 	// Load shaders
 	std::array<VkPipelineShaderStageCreateInfo, 2> shader_stages;
-
```

Comment:
This doesn't seem intentional. A single blank line has been removed.

Reply:
Fixed